joss_paper/paper.md

```diff
@@ -0,0 +1,167 @@
+authors:
+  - name: Daniel F. Ryan^[co-first author] # note this makes a footnote saying 'co-first author'
+    orcid: 0000-0001-8661-3825
+    affiliation: 1 # (Multiple affiliations must be quoted)
+  - name: Stuart Mumford^[co-first author] # note this makes a footnote saying 'co-first author'
+    affiliation: 2
+  - name: Author with no affiliation^[corresponding author]
+    affiliation: 3
+affiliations:
+ - name: University of Applied Sciences Northwest Switzerland
+   index: 1
+ - name: University of Sheffield
+   index: 2
+date: 11 December 2021
+bibliography: paper.bib
+
+---
+
+# Summary
+
+ndcube is a free, open-source, community-developed Python package whose purpose
+is to faciliate the manipulation and visualization of astronomical data.
+It does this by linking the data and coordinates in single data objects.
+These objects can be manipulated via array-like slicing operations which modify
+the data and coordinates simultaneously.
+They also allow coordinate transformations to be performed with reference to the
+shape of the data and enable coordinate-aware visualizations.
+This data-coordinate coupling allows users to analyze their data more
+easily and accurately, thus helping to boost their scientific output.
+
+ndcube is agnostic to the physical properties represented by the data values and axes.
+This makes it a powerful base upon which to build tools for specific types of data.
+This might be a specific number and/or combination of physical types
+(spectrograms, image cubes, etc.), or data from specific instruments or simulations.
+Thus, ndcube can also enhance the productivity of developers by centralizing the
+development and maintenance of the most useful and general functionalities.
+This leaves more time for developing a greater range of tools for the community
+and/or enables part-time developers to devote more effort to other aspects of their jobs,
+e.g. scientific analysis.
+
+A network benefit of ndcube is that it standardizes the APIs for handling N-D data.
+Broad adoption throughout astronomy and heliophysics would help scientists to more easily
+work with data from different sub-fields.
+This can help facilitate synergies between new combinations of data,
+foster inter-field collaborations, and promote scientific innovation.
+
+# Statement of Need
+
+N-dimensional data sets are common in all areas of science and beyond.
+For example, a series of images taken sequentially with a CCD camera can be stored
+as a single 3-D array with two spatial axes and one temporal axis.
+The value in each array-element can represent the reading in a pixel at a given time.
+In astronomy, the relationship between the array element and the location and time
+in the Universe being observed is often represented by the World Coordinate System (WCS) framework.
+WCS’s ability to handle many different physical types (e.g. spatial, temporal, spectral, etc.)
+and projections (e.g. RA and Dec., helioprojective latitude and longitude, etc.)
```

Comment:
We should mention this is a *functional* framework where the transformation is modelled and then evaluated on demand, vs lookup tables.